library/src/helpers/__tests__/bemClasses.test.ts

```diff
@@ -64,4 +65,18 @@ describe('bemClasses', () => {
       expect(bemClasses.concatenate(arr)).toBe(expectedClassName);
     });
   });
+
+  describe('identifier', () => {
```

Comment:
write a bit more tests for this function

Reply:
Four elements in the input array test all cases. See in code.